docs/docs/quickstart/standalone.md

```diff
@@ -30,44 +34,67 @@ git clone https://github.com/star-whale/starwhale.git
 cd starwhale
 ```
 
-If [git-lfs](https://git-lfs.github.com/) has not been previously installed in the local environment(the command is `git lfs install`), you need to download the trained model file.
-
-```bash
-wget https://media.githubusercontent.com/media/star-whale/starwhale/main/example/mnist/models/mnist_cnn.pt -O example/mnist/models/mnist_cnn.pt
-```
-
 We will use ML/DL HelloWorld code `MNIST` to start your Starwhale journey. The following steps are all performed in the `starwhale` directory.
 
-![Core Workflow](../img/standalone-core-workflow.gif)
-
 ## 3. Building Runtime
 
 Runtime example code are in the `example/runtime/pytorch` directory.
 
-- Build the Starwhale Runtime bundle:
+- Build the Starwhale Runtime bundle:.
+
+  :::tip
+  When you first build runtime, creating an isolated python environment and downloading python dependencies will take a lot of time. The command execution time is related to the network environment of the machine and the number of packages in the runtime.yaml. Using the befitting pypi mirror and cache config in the `~/.pip/pip.conf` file is a recommended practice.
+
+  For users in the mainland of China, the following conf file is an option:
+
+    ```conf
+    [global]
+    cache-dir = ~/.cache/pip
+    index-url = https://mirrors.aliyun.com/pypi/simple/
+    extra-index-url = https://pypi.doubanio.com/simple
+
+    [install]
+    trusted-host=pypi.doubanio.com mirrors.aliyun.com
+    ```
+
+  :::
 
   ```bash
-  swcli runtime build .
+  swcli runtime build example/runtime/pytorch
   ```
 
-- Check your local Starwhale Runtimes:
+- Check your local Starwhale Runtime:
 
   ```bash
   swcli runtime list
   swcli runtime info pytorch/version/latest
   ```
 
+- Restore Starwhale Runtime(optional):
+
+  ```bash
+  swcli runtime restore pytorch/version/latest
+  ```
+
 ## 4. Building Model
 
 Model example code are in the `example/mnist` directory.
 
-- Build a Starwhale Model:
+- Download pre-trained model file:
+
+  ```bash
+  cd example/mnist
+  make download-model
```

Comment:
Add comments for users in mainland of China?  
```sh
CN=1 make download-model
```

Reply:
ok